hubspot/connection/__init__.py

```diff
@@ -228,11 +230,11 @@ def __exit__(self, exc_type, exc_value, traceback):
         self._session.close()
 
 
-_AuthenticationKey = Record.create_type('_AuthenticationKey', 'key_value')
+_AuthenticationKey = Record.create_type(str('_AuthenticationKey'), str('key_value'))
```

Comment:
That error is very strange. I'm the author of PyRecord, and the test I wrote for that case works on Python 2.7/3.3/3.4/3.5: https://github.com/gnarea/pyrecord/blob/11aa1625b810aa025d3c0edd3f2c04375d5e46b0/tests/test_record_type.py#L25

I'd expect the expression `Record.create_type('AuthenticationKey', 'key_value')` to work on Python 2 and 3. I'll investigate that, but for now I'd suggest to remove these `str()` calls from here as I could deal with that case in PyRecord if necessary.